kodilla-cucumber/src/main/java/wallet/Teller.java

```diff
@@ -8,11 +8,13 @@ public Teller(CashSlot cashSlot) {
         this.cashSlot = cashSlot;
     }
 
-    public int withdraw(Wallet wallet, int amount) {
-        if (amount > 0) {
+    public String withdraw(Wallet wallet, int amount) {
+        if (wallet.getBalance() >= amount) {
             int debit = wallet.debit(amount);
             cashSlot.dispense(debit);
+            return "Cash withdrawn";
+        } else {
+            return "I have insufficient funds in my account";
```

Comment:
czemu zwracamy String? :O

Reply:
bo mamy zwrócić Stringa w ostatnim scenariuszu Feature: Prevent users from going overdrawn, jest napisane:     And I should be told that I have insufficient funds in my account